spec/rails_helper.rb

```diff
@@ -41,9 +41,9 @@
 Capybara.register_driver :selenium do |app|
   Capybara::Selenium::Driver.new(app, browser: :chrome)
 end
-Capybara.javascript_driver = :selenium                 # Run feature specs with Firefox
+# Capybara.javascript_driver = :selenium                 # Run feature specs with Firefox
 # Capybara.javascript_driver = :selenium_chrome          # Run feature specs with Chrome
-# Capybara.javascript_driver = :selenium_chrome_headless # Run feature specs with headless Chrome
+Capybara.javascript_driver = :selenium_chrome_headless # Run feature specs with headless Chrome
```

Comment:
what do these changes get us? 
are they required?

Reply:
Sorry, should have commented on this change.

This reverses a change you made in a [recent PR](https://github.com/GSA/touchpoints/pull/1613/files#diff-053aa3ea46cb7c6a649e0d0fc592ce729b66e1d44be475516fd58857ba695862). I was guessing that you changed from headless to headed browser for debugging purposes and didn't mean to commit it. I noticed the change because now, when I run rspec locally, the headed browser being used for testing basically takes over the input devices on my computer and I can't type or use the keyboard.

Reply:
🤦‍♂️

👍